apps/taiga-lumbermill/src/dashboards/register/register.component.ts

```diff
@@ -0,0 +1,106 @@
+import {CommonModule} from '@angular/common';
+import {ChangeDetectionStrategy, Component, inject} from '@angular/core';
+import {toSignal} from '@angular/core/rxjs-interop';
+import type {AbstractControl, ValidationErrors, ValidatorFn} from '@angular/forms';
+import {
+    FormControl,
+    FormGroup,
+    FormsModule,
+    ReactiveFormsModule,
+    Validators,
+} from '@angular/forms';
+import {Router} from '@angular/router';
+// import {TUI_FALSE_HANDLER} from '@taiga-ui/cdk';
+import {
+    TuiAppearance,
+    TuiButton,
+    TuiError,
+    TuiLabel,
+    TuiLink,
+    TuiLoader,
+    TuiTitle,
+} from '@taiga-ui/core';
+import {TUI_VALIDATION_ERRORS, TuiCheckbox, TuiFieldErrorPipe} from '@taiga-ui/kit';
+import {TuiCardLarge, TuiHeader} from '@taiga-ui/layout';
+import {TuiInputModule, TuiInputPasswordModule} from '@taiga-ui/legacy';
+import {map, startWith, Subject, switchMap, timer} from 'rxjs';
+
+export const checkPasswords: ValidatorFn = (
+    control: AbstractControl,
+): ValidationErrors | null => {
+    const name = control.get('password');
+    const role = control.get('passwordAgain');
+
+    return name && role && name.value === role.value ? {unambiguousRole: true} : null;
```

Comment:
It looks like "unambiguous role" is a copy-paste from a doc. Name it according to its purpose.